### PR TITLE
Add smoke test for mount_flags_to_string()

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/test/test-mount-util.c [hunk 1]
```diff
@@ -78,6 +79,50 @@ static void test_mount_option_mangle(void) {
...
+        test_mount_flags_to_string_one(MS_SLAVE, "MS_SLAVE");
+        test_mount_flags_to_string_one(MS_SHARED, "MS_SHARED");
+        test_mount_flags_to_string_one(MS_RELATIME, "MS_RELATIME");
+        test_mount_flags_to_string_one(MS_KERNMOUNT, "MS_KERNMOUNT");
+        test_mount_flags_to_string_one(MS_I_VERSION, "MS_I_VERSION");
+        test_mount_flags_to_string_one(MS_STRICTATIME, "MS_STRICTATIME");
+        test_mount_flags_to_string_one(MS_LAZYTIME, "MS_LAZYTIME");
+        test_mount_flags_to_string_one(MS_LAZYTIME|MS_STRICTATIME, "MS_STRICTATIME|MS_LAZYTIME");
+        test_mount_flags_to_string_one(~0UL,
+                                       "MS_RDONLY|MS_NOSUID|MS_NODEV|MS_NOEXEC|MS_SYNCHRONOUS|MS_REMOUNT|MS_MANDLOCK|MS_DIRSYNC|MS_NOSYMFOLLOW|MS_NOATIME|MS_NODIRATIME|MS_BIND|MS_MOVE|MS_REC|MS_SILENT|MS_POSIXACL|MS_UNBINDABLE|MS_PRIVATE|MS_SLAVE|MS_SHARED|MS_RELATIME|MS_KERNMOUNT|MS_I_VERSION|MS_STRICTATIME|MS_LAZYTIME|fffffffffc000200");
```
Comment (bluca) [inline src/test/test-mount-util.c]: if the kernel adds a new flag, will this break?

**Comment by yuwata** [comment]
LGTM.

File: src/test/test-mount-util.c (hunk 1 above)
Comment (yuwata) [reply src/test/test-mount-util.c]: `mount_flags_to_string()` stringifies only known flags. So, the test still OK when kernel adds a new flag.

**Comment by yuwata** [comment]
bionic-i386 fails with the following:
```
/* test_mount_flags_to_string */
flags: 0 → 0/"0"
flags: 0X1 → 0/"MS_RDONLY"
flags: 0X2 → 0/"MS_NOSUID"
flags: 0X4 → 0/"MS_NODEV"
flags: 0X8 → 0/"MS_NOEXEC"
flags: 0X10 → 0/"MS_SYNCHRONOUS"
flags: 0X20 → 0/"MS_REMOUNT"
flags: 0X40 → 0/"MS_MANDLOCK"
flags: 0X80 → 0/"MS_DIRSYNC"
flags: 0X100 → 0/"MS_NOSYMFOLLOW"
flags: 0X400 → 0/"MS_NOATIME"
flags: 0X800 → 0/"MS_NODIRATIME"
flags: 0X1000 → 0/"MS_BIND"
flags: 0X2000 → 0/"MS_MOVE"
flags: 0X4000 → 0/"MS_REC"
flags: 0X8000 → 0/"MS_SILENT"
flags: 0X10000 → 0/"MS_POSIXACL"
flags: 0X20000 → 0/"MS_UNBINDABLE"
flags: 0X40000 → 0/"MS_PRIVATE"
flags: 0X80000 → 0/"MS_SLAVE"
flags: 0X100000 → 0/"MS_SHARED"
flags: 0X200000 → 0/"MS_RELATIME"
flags: 0X400000 → 0/"MS_KERNMOUNT"
flags: 0X800000 → 0/"MS_I_VERSION"
flags: 0X1000000 → 0/"MS_STRICTATIME"
flags: 0X2000000 → 0/"MS_LAZYTIME"
flags: 0X3000000 → 0/"MS_STRICTATIME|MS_LAZYTIME"
flags: 0XFFFFFFFF → 0/"MS_RDONLY|MS_NOSUID|MS_NODEV|MS_NOEXEC|MS_SYNCHRONOUS|MS_REMOUNT|MS_MANDLOCK|MS_DIRSYNC|MS_NOSYMFOLLOW|MS_NOATIME|MS_NODIRATIME|MS_BIND|MS_MOVE|MS_REC|MS_SILENT|MS_POSIXACL|MS_UNBINDABLE|MS_PRIVATE|MS_SLAVE|MS_SHARED|MS_RELATIME|MS_KERNMOUNT|MS_I_VERSION|MS_STRICTATIME|MS_LAZYTIME|fc000200"
Assertion 'streq(x, expected)' failed at src/test/test-mount-util.c:89, function test_mount_flags_to_string_one(). Aborting.
```

**Comment by yuwata** [comment]
Force-pushed a fixed version.

**Comment by keszybz** [comment]
bionic-ppc64el failed with localed-locale       FAIL non-zero exit status 1, unrelated.

**Pull request merged**